src/scripts/main.js

```diff
@@ -1,11 +1,22 @@
 'use strict';
 
 function waitFor(element, eventName) {
-// write your code here
+  const promise1 = (resolve, reject) => {
```

Comment:
This can be simplify to be just:

```suggestion
  return new Promise((resolve, reject) => {
```